maintainers/maintainer-list.nix

```diff
@@ -970,6 +970,12 @@
     email = "sivaraman.balaji@gmail.com";
     name = "Balaji Sivaraman";
   };
+  baloo = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add baloo``.